Utilities/DatabasePorter/api.go

```diff
@@ -130,7 +130,7 @@ func GetEntry(hash string) (interfaces.IEBEntry, error) {
 		}
 		return entry, nil
```

Comment:
There is no loop here.... You can't try 10 times because you will always return, right?